src/AppBundle/Form/DataTransformer/TagArrayToStringTransformer.php

```diff
@@ -54,7 +54,7 @@ public function reverseTransform($string)
             return [];
         }
 
-        $names = explode(',', $string);
+        $names = array_filter(array_unique(array_map('trim', explode(',', $string))));
```

Comment:
The tagsinput plugins itself does not allows duplicated, empty tags, extra spaces, double comma, BUT it could be hacked :) so I'm :+1: to check this on server side as double validation. Maybe I would add `strtolower` too?